perception/lidar_centerpoint/launch/lidar_centerpoint.launch.xml

```diff
@@ -7,24 +7,11 @@
   <arg name="model_path" default="$(var data_path)/lidar_centerpoint"/>
   <arg name="model_param_path" default="$(find-pkg-share lidar_centerpoint)/config/$(var model_name).param.yaml"/>
   <arg name="class_remapper_param_path" default="$(find-pkg-share lidar_centerpoint)/config/detection_class_remapper.param.yaml"/>
-  <arg name="score_threshold" default="0.35"/>
-  <arg name="has_twist" default="false"/>
-  <arg name="build_only" default="false" description="shutdown node after TensorRT engine file is built"/>
```

Comment:
[MUST] Could you please retain the `build_only` parameters in the launch file? Since it is NOT a node parameter, it is used for controlling the node from outside.

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/6167/commits/1c74fe4c0e973ce7ffc252e319ddbc992aa6ef00
Also added a description in readme as well: https://github.com/autowarefoundation/autoware.universe/pull/6167/commits/b028c8ad909d6ee49a7a27260e3727e6de9f5abd